lib/src/main/kotlin/com/lemonappdev/konsist/core/util/HyperlinkUtil.kt

```diff
@@ -0,0 +1,10 @@
+package com.lemonappdev.konsist.core.util
+
+import java.io.File
+
+object HyperlinkUtil {
+    fun toHyperlink(path: String): String {
+        val absolutePath = File(path).absolutePath
+        return "file://$absolutePath"
```

Comment:
It looks like we don't need `file://` as IDE can make clickable link without it
<img width="518" alt="image" src="https://github.com/user-attachments/assets/b804b472-07b4-4c91-bc6c-fa82d76630a8">


<img width="592" alt="image" src="https://github.com/user-attachments/assets/6c9af787-8dd0-430b-ba2d-826f0683f8e0">

Reply:
1. Also let's change format

From:
`file:///Users/igorwojda/IdeaProjects/konsist/samples/starter-projects/android-gradle-kotlin-junit-5/app/src/main/kotlin/com/sample/MainActivity.kt:14:1 (Class 'MainActivity')`

To:
`Class MainActivity file:///Users/igorwojda/IdeaProjects/konsist/samples/starter-projects/android-gradle-kotlin-junit-5/app/src/main/kotlin/com/sample/MainActivity.kt:14:1`

2. Since name is uppercase here let's update names in `AsciiTreeNode` class (`file` -> `File`, `import` -> `Import`)

Reply:
1. and 2. resolved